pagopa/io-app
# [IOCOM-133] Replaces custom PN activation decoder with autogenerated one

## Short description
This PR replaces a custom decoder used in the PN activation service with the autogenerated one

## List of changes proposed in this pull request
- `ts/features/pn/api/backendPn.ts`: `upsertPNActivationDecoderCustom` has been removed and replaced with `upsertPNActivationDefaultDecoder` (`definitions/pn/requestTypes`)

## How to test
Using the dev server, configure it in order to enable PN (`includePn: true` in the `config` file).
In the application, navigate to the National services and select the PN one ("_Avvisi di Cortesia_" under "_Piattaforma delle notifiche digitali_").
At the bottom of the service details page, there should be a button that allows to enable or disable the service. Check that the API call is received by the dev server (`POST /api/v1/pn/activation`) and the service page is then updated with the proper enabled/disabled value (and that subsequent usages of such buttons are consistent with its state)
